evm/computation.py

```diff
@@ -448,7 +473,7 @@ def apply_computation(cls, vm_state, message, transaction_context):
     # Opcode API
     #
     @property
-    def precompiles(self):
+    def precompiles(self) -> Dict[bytes, Callable]:
```

Comment:
I think this can be made more specific as `Callable[[Computation], None]` or `Callable[[Computation], Any]`.  I think the later with `Any` is fine to **allow** opcodes to return values but not require any kind of return type or value.

Reply:
Good point. Updated the PR. There are actually two other annotations in here that are annotated as just `Tuple` that could probably be more specific. I haven't had time to follow up on them yet so if you know the types from the top of your head I'm happy to update them as well.

If not we can either:

- keep this PR open until I updated them
- merge now and update them with a subsequent PR

The cool thing is that as more and more type annotations get added it becomes easier and easier for `mypy` to assist you. I already updated a couple of type annotations that were previously wrong because as you put in more parts of the puzzle `mypy` becomes stronger and stronger to infer the correct types and eventually things converge to the correct types.

Reply:
Ok, I updated the two APIs that where previously typed to return just `Tuple` to now be.

```
def get_accounts_for_deletion(self) -> Tuple[Tuple[bytes, bytes], ...]:
```

and 

```
def get_log_entries(self) -> Tuple[Tuple[bytes, List[int], bytes], ...]:
```